app/models/controlled_work_document_content.rb

```diff
@@ -154,15 +170,23 @@ def combined_childcare_costs
       (session_data.dig("api_response", "assessment", "partner_disposable_income", "childcare_allowance") || 0)
   end
 
-  def additional_non_smod_properties_sum(attribute)
-    additional_non_smod_properties.sum { _1[attribute] }
+  def additional_properties_sum(attribute)
+    additional_properties.sum { _1[attribute] }
   end
 
-  def additional_non_smod_properties
-    @additional_non_smod_properties ||= [
-      (session_data.dig("api_response", "assessment", "capital", "capital_items", "properties", "additional_properties") unless additional_property_in_dispute?),
-      session_data.dig("api_response", "assessment", "partner_capital", "capital_items", "properties", "additional_properties"),
-    ].flatten.compact
+  def additional_properties
+    case form_type
+    when "cw1"
```

Comment:
I really don't think this model, which is just a wrapper for some session data and until now knew nothing about CW forms, should have to know about the logic associated with specific forms. A method that returns different output depending on who is calling it is a bit iffy anyway, and this would mean committing to updating this method for every new CW form we add.

Hitherto the yaml file has been the only place that knows about the fields of any given CW form. I'd be strongly inclined to keep it that way.